docker/image/console/root/usr/lib/task/database/available.sh

```diff
@@ -25,6 +25,10 @@ function task_database_available()
             exit 1
         fi
 
+        if ! (( counter % 5 )); then
+          echo "Waiting for database to become available: '${command}'"
```

Comment:
Careful about printing the database password out for mysql. Maybe direct to stderr too?

Reply:
```suggestion
          echo "Waiting for database '$DB_HOST' to become available" >&2
```